docs/content/en/docs/Concepts/mqtt/index.md

```diff
@@ -449,4 +449,11 @@ Under this topic a message should be sent whenever an assembly at a certain stat
 }
 ```
 
+### /TimescaleDB structure
```

Comment:
This should be 
`## Database model`

or

`## TimescaleDB structure`

(without one # and one /)